controllers/main_controller.rb

```diff
@@ -9,3 +11,15 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/:num" do
+  Website = Segmentation.new
```

Comment:
In Ruby, don't capitalize variables. Capitalized keywords are classes. So this should be `website = Segmentation.new`.